src/utils/FetchConfig.ts

```diff
@@ -22,89 +22,89 @@ export async function FetchConfig (): Promise<any> {
 
   const registriesSearchApiUrl: string =
-    (process.env.VUE_APP_REGISTRIES_SEARCH_API_URL + process.env.VUE_APP_REGISTRIES_SEARCH_API_VERSION + '/')
+    (import.meta.env.VUE_APP_REGISTRIES_SEARCH_API_URL + import.meta.env.VUE_APP_REGISTRIES_SEARCH_API_VERSION + '/')
   sessionStorage.setItem('REGISTRIES_SEARCH_API_URL', registriesSearchApiUrl)
 
-  const businessApiKey: string = process.env.VUE_APP_REGISTRIES_SEARCH_API_KEY
+  const businessApiKey: string = import.meta.env.VUE_APP_REGISTRIES_SEARCH_API_KEY
   sessionStorage.setItem('BUSINESS_API_KEY', businessApiKey)
 
-  const authApiUrl: string = (process.env.VUE_APP_AUTH_API_URL + process.env.VUE_APP_AUTH_API_VERSION + '/')
+  const authApiUrl: string = (import.meta.env.VUE_APP_AUTH_API_URL + import.meta.env.VUE_APP_AUTH_API_VERSION + '/')
   sessionStorage.setItem('AUTH_API_URL', authApiUrl)
 
-  const payApiUrl: string = (process.env.VUE_APP_PAY_API_URL + process.env.VUE_APP_PAY_API_VERSION + '/')
+  const payApiUrl: string = (import.meta.env.VUE_APP_PAY_API_URL + import.meta.env.VUE_APP_PAY_API_VERSION + '/')
   sessionStorage.setItem('PAY_API_URL', payApiUrl)
 
   // for system alert banner (sbc-common-components)
-  const statusApiUrl: string = (process.env.VUE_APP_STATUS_API_URL + process.env.VUE_APP_STATUS_API_VERSION)
+  const statusApiUrl: string = (import.meta.env.VUE_APP_STATUS_API_URL + import.meta.env.VUE_APP_STATUS_API_VERSION)
   sessionStorage.setItem('STATUS_API_URL', statusApiUrl)
 
-  const siteminderLogoutUrl: string = process.env.VUE_APP_SITEMINDER_LOGOUT_URL
+  const siteminderLogoutUrl: string = import.meta.env.VUE_APP_SITEMINDER_LOGOUT_URL
   if (siteminderLogoutUrl) {
     sessionStorage.setItem('SITEMINDER_LOGOUT_URL', siteminderLogoutUrl)
   }
 
-  const keycloakAuthUrl: string = process.env.VUE_APP_KEYCLOAK_AUTH_URL;
+  const keycloakAuthUrl: string = import.meta.env.VUE_APP_KEYCLOAK_AUTH_URL;
   (<any>window).keycloakAuthUrl = keycloakAuthUrl
 
-  const keycloakRealm: string = process.env.VUE_APP_KEYCLOAK_REALM;
+  const keycloakRealm: string = import.meta.env.VUE_APP_KEYCLOAK_REALM;
   (<any>window).keycloakRealm = keycloakRealm
 
-  const keycloakClientId: string = process.env.VUE_APP_KEYCLOAK_CLIENTID;
+  const keycloakClientId: string = import.meta.env.VUE_APP_KEYCLOAK_CLIENTID;
   (<any>window).keycloakClientId = keycloakClientId
 
-  const iaSurveyId: string = process.env.VUE_APP_IA_SURVEY_ID
+  const iaSurveyId: string = import.meta.env.VUE_APP_IA_SURVEY_ID
   // NB: set empty string if iaSurveyId is falsy (undefined, null or 0)
   sessionStorage.setItem('IA_SURVEY_ID', iaSurveyId || '')
 
-  const hotjarId: string = process.env.VUE_APP_HOTJAR_ID;
+  const hotjarId: string = import.meta.env.VUE_APP_HOTJAR_ID;
   (<any>window).hotjarId = hotjarId
 
-  const addressCompleteKey: string = process.env.VUE_APP_ADDRESS_COMPLETE_KEY;
+  const addressCompleteKey: string = import.meta.env.VUE_APP_ADDRESS_COMPLETE_KEY;
```

Comment:
Check if this is set in the temporary URL.

Reply:
I think this will _probably_ work in Dev. I _think_ there's GCP config needed to set the keys from 1Password vaults and that may be what's missing for the temp URL.

If we break Dev, though, I don't know how to fix it. (I don't have permissions to the GCP config.) @seeker25 , any ideas? We could also reach out to Thor.

Reply:
Sev, I was thinking. Can I create a branch on the main repo and merge to that branch? After doing that, I'll deploy manually the branch to DEV just to test and make sure everything is OK. If all is well, I can merge to main after that.

I don't want to break DEV like you mentioned since it might not be possible to find a quick fix for whatever broke it. Thoughts?

Reply:
Great idea. LMK if you need help with that.

Reply:
I honestly think a team lead should have access to GCP config - we'll have to talk to Patrick about it when he gets back off of vacation

Reply:
I think that's planned, but it's a few weeks away.